scripts/import_cloudwatch_log_groups.py

```diff
@@ -0,0 +1,83 @@
+import json
+
+import boto3
+from more_itertools import (
+    one,
+)
+
+from azul import (
+    config,
+    logging,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+from azul.terraform import (
+    terraform,
+)
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    def add_prefix(name):
+        return 'aws_cloudwatch_log_group.' + name
+
+    log_groups = {}
+    tf_component = config.terraform_component
+
+    if tf_component == 'shared':
+        if config.slack_integration:
+            # Chatbot log groups
+            name = add_prefix('chatbot')
+            log_group = '/aws/chatbot/' + config.qualified_resource_name('chatbot')
+            log_groups[name] = log_group
+    elif not tf_component:
+        # Lambda function log groups
+        tf_path = f'{config.project_root}/terraform/api_gateway.tf.json'
+        with open(tf_path, 'r') as f:
+            tf_json = json.load(f)
+        for resources_by_type in tf_json['resource']:
+            resource_type, resources = one(resources_by_type.items())
+            if resource_type == 'aws_lambda_function':
+                for resource in resources:
+                    for resource_name, resource_def in resource.items():
+                        name = add_prefix(resource_name + '_lambda')
+                        log_group = '/aws/lambda/' + resource_def['function_name']
+                        log_groups[name] = log_group
+        # API Gateway execution log groups
+        # For these we have to query AWS to get the API Gateway IDs.
+        api_client = boto3.client('apigateway')
+        paginator = api_client.get_paginator('get_rest_apis')
+        for api_page in paginator.paginate():
+            for api in api_page['items']:
+                name, stage = config.unqualified_resource_name(api['name'])
+                if stage == config.deployment_stage:
+                    name = f'{name}_api_execution'
+                    log_group = f"API-Gateway-Execution-Logs_{api['id']}/{stage}"
+                    log_groups[add_prefix(name)] = log_group
+    else:
+        pass
+
+    log_client = boto3.client('logs')
+    paginator = log_client.get_paginator('describe_log_groups')
+    existing_log_groups = [
```

Comment:
A set might be more efficient here
```suggestion
    existing_log_groups = {
```